packages/formation/sass/modules/_m-print.scss

```diff
@@ -8,19 +8,24 @@
   header, footer, nav {
     display: none;
   }
-  
+
   .schemaform-title,
   .schemaform-subtitle,
   .screen-only {
     display: none;
   }
-  
+
   .print-only {
     display: block;
   }
-  
-  /* Include href when printing */
-  a:not([href^="tel"]):after {
-    content: " (https://va.gov" attr(href) ")";
+
+  /* Relative URLs - include the HREF with the domain prepended */
+  a[href^="/"]:after {
+    content: " (https://www.va.gov" attr(href) ")";
```

Comment:
As shown in the diff, we previously omitted the `www`. This doesn't seem right to me it - it only saves four characters, so it's not as though we save a bunch of space on the page, and creates an unnecessary redirect. It also just doesn't look right to me to leave out the `www`.